test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/NestedJSClassTest.scala

```diff
@@ -900,6 +906,8 @@ object NestedJSClassTest {
 
   class DynamicInnerClass_Issue4402(arg: String) extends js.Object {
```

Comment:
It's a bit ugly that we have issue4402 here in the name but not issue4801. Maybe it makes sense to just comment how the test tests the relevant issues?

Reply:
I was not sure what's best to do here. I added `_Issue4801` to the method name, `methodSuper_Issue4801`, as it is effectively that method that tests this issue. Does that make sense?

Reply:
SG